### DISCUSSION TITLE
Clicking Play Button Doesnt Play on Safari

### DISCUSSION THREAD

**Discussion by jtan80813** [cookpete/react-player#1786] [General]
Please check this

https://stackoverflow.com/questions/66380923/clicking-video-doesnt-play-in-ios-in-react?noredirect=1#comment117360790_66380923

**Comment by gordielachance** [comment]
post does not exists on stackoverflow.

**Comment by jtan80813** [comment]
@gordielachance It does exist. Pls check again

**Comment by jtan80813** [comment]
@reynoldsnlp Why? Its not yet resolved
